Python/bin/packUtil.py

```diff
@@ -1060,8 +1061,10 @@ def packTupleID(triplet):
          tmp = m.group(1)
          #print ("m.groups is ", m.groups())
          if len(tmp) > 7:
-            raise RuntimeError("Can't pack " + trkSub + " because it it is too long")
-         packedTrkSub = m.group(1)
+            print('Warning: trkSub ' + trkSub + ' is too long for obs80, removing the first character')
```

Comment:
I think we should be careful about printing to `stdout` since we have converted the python tools to stdout/stdin stream utilities. You can change this to
```
print('Warning: trkSub ' + trkSub + ' is too long for obs80, removing the first character', file=sys.stderr)
```
to print the warning to stderr (where it might be better placed anyway).

Reply:
@stevenstetzler you're right. I forgot about that. I fixed it.